build.sh

```diff
@@ -57,4 +57,6 @@ fi
 
 [ "${PORTS_MICROPYTHON}" = "y" ] && ./phoenix-rtos-ports/micropython/build.sh
 
+[ "X${PORTS_SSCEP}" == "Xy" ] && ./phoenix-rtos-ports/sscep/build.sh
```

Comment:
```suggestion
[ "${PORTS_SSCEP}" = "y" ] && ./phoenix-rtos-ports/sscep/build.sh
```
similar to the above